ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
Similarly, some of the error messages say things like "it's an array element", "it's a lazy expression" or "it's a polymorphic variant".  But they are talking about types, not terms - should they say "it's the type of ..." instead?  Certainly we use this kind of shorthand all the time, so I don't feel particularly strongly.

Reply:
And here's another case where I think there's a kind of category error:
```
1 | module type S = sig val x : t_void end
                                ^^^^^^
Error: This type signature for x is not a value type.
       The layout of x is void, because
         of the definition of t_void at line 6, characters 0-19.
       But the layout of x must be a sublayout of value, because
         it's stored in a module structure.
```
We say "the layout of `x` is void", but `x` isn't a type.  Again we use this short hand all the time, but I guess I lean in the direction that errors should be precise.